README.rst

```diff
@@ -1,4 +1,65 @@
 Snapcraft rocks
 ===============
 
-TODO
+This repository contains sources to generate `Snapcraft`_ `rocks`_. You can use
+these sources to create OCI-compliant container images that run specific
+versions of Snapcraft to create snaps.
+
+The sources in this repository are capable of building different versions of
+Snapcraft, targeting snaps for different cores. The sources are separated
+into Git branches in the form ``coreX-Y``, where ``X`` is a core number (like
+``22`` or ``24``), and ``Y`` is a major version of Snapcraft (like ``7`` or
+``8``).
+
+For example, branch ``core22-7`` contains the Rockcraft project to create
+a rock containing the latest stable version of Snapcraft 7 and can be used
+to create snaps targetting ``core22``.
+
+Supported versions
+------------------
+
+These are the currently supported combinations of Snapcraft versions and cores:
+
+- Branch ``core22-7``, for rocks with latest Snapcraft 7 to build ``core22``
+  snaps.
+
+
+Requirements
+------------
+
+- `Rockcraft`_ is needed to generate the rocks (minimum version ``1.1.0``).
+- `Docker`_ or some other engine is needed to create containers from the images.
```

Comment:
Can you add a disclaimer and/or link about how docker may break network for LXD?

 https://canonical-craft-providers.readthedocs-hosted.com/en/latest/explanation/#failure-to-properly-execute-commands-that-depend-on-network-access ?